src/mpid/ch4/src/ch4_types.h

```diff
@@ -14,7 +14,11 @@
 #include <mpidimpl.h>
 #include <stdio.h>
 #include "mpir_cvars.h"
+#ifdef USE_PMI2_API
```

Comment:
Yes. However, no harm leaving there too. 
Can you please do that? I am leaving out of town today.